tests/conftest.py

```diff
@@ -1,7 +1,11 @@
 import tempfile
+from pathlib import Path
```

Comment:
I understand that using `pathlib` is a better choice than `os.path` nowadays but I'd prefer to have a consistent usage in this project. Does anything speak using `os.path` in this pull request and switch the entire project to `pathlib` in a separate pull request?

Reply:
I vastly prefer the usage of `pathlib` (for numerous reasons) but if consistency is a priority we can switch to `os.path` for the time being.